src/main/Lock.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { faLock } from "@fortawesome/free-solid-svg-icons";
+
+import { settings } from "../config";
+import { useDispatch, useSelector } from "react-redux";
+import { setError } from "../redux/errorSlice";
+import { LockData, setLock, video } from "../redux/videoSlice";
+import { client } from "../util/client";
+
+export type ILock = {
+  lockingActive: boolean;
+  lockRefresh: number;
+  lockState: boolean;
+  lock: LockData;
+};
+
+const Lock: React.FC<{}> = () => {
+  const [state, setState] = useState({
```

Comment:
Also goes unused, since all these values are stored in redux. Respectively, all "setState" calls are superfluous.